main.go

```diff
@@ -191,9 +197,8 @@ func main() {
 			log.Printf("received event of type: %T", event)
 		}
 
-		err := renameAllWs()
-		if err != nil {
-			log.Print(err)
+		if err := renameAllWs(); err != nil {
+			log.Fatal(err)
```

Comment:
We could let this error pass. But should we? I think it's fair to crash on error (or maybe reset the names and then crash?).

Reply:
Seems fair. We don't really reset names anywhere else, so this is fine.